source/robot_model/include/robot_model/Model.hpp

```diff
@@ -303,20 +289,6 @@ inline void Model::set_gravity_vector(const Eigen::Vector3d& gravity) {
   this->robot_model_.gravity.linear(gravity);
 }
 
-inline state_representation::CartesianPose Model::forward_geometry(const state_representation::JointState& joint_state,
```

Comment:
This change is nice in my opinion, although it's unrelated to the PR. It's small enough that it's fine though.